src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -168,5 +168,21 @@ public int MaxModelValidationErrors
         /// <see langword="false"/> by default.
         /// </summary>
         public bool AllowBindingUndefinedValueToEnumType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the flag to determine if model binding should return a response with HTTP 400 Bad Request for
+        /// all exceptions(including ones not related to bad input) that occur during deserialization in <see cref="IInputFormatter"/>s.
+        /// This flag applies only to custom <see cref="IInputFormatter"/>s. For modifying the behavior of in-built
+        /// <see cref="IInputFormatter"/>s, see <see cref="SendBadRequestForAllExceptionsDuringDeserializationInInbuiltFormatters"/>.
+        /// Defaults is <see langword="true"/>.
+        /// </summary>
+        public bool SendBadRequestForAllExceptionsDuringInputFormatterDeserialization { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets the flag to determine if model binding should return a response with HTTP 400 Bad Request for
```

Comment:
Model binders don't ever 'return 400'. The question here is whether model binders should handle exceptions other than InputFormatterException